content/spin/cli-reference.md

```diff
@@ -3315,26 +3288,39 @@ OPTIONS:
 
 {{ blockEnd }}
 
-{{ blockEnd }}
+{{ startTab "v1.3"}}
+
+<!-- @selectiveCpy -->
+
+```console
+$ spin watch --help
 
-`spin watch` relies on configuration in your application manifest to know what files it should watch. For each component you should set the `component.build.watch` parameter with a list of glob patterns that your source files will match:
+spin-watch 
+Build and run the Spin application, rebuilding and restarting it when files change
+
+USAGE:
+    spin watch [OPTIONS] [UP_ARGS]...
+
+ARGS:
+    <UP_ARGS>...    Arguments to be passed through to spin up
 
-```toml
-[component.build]
-# Example watch configuration for a Rust application
-watch = ["src/**/*.rs", "Cargo.toml"]
+OPTIONS:
+    -c, --clear                       Clear the screen before each run
+    -d, --debounce <DEBOUNCE>         Set the timeout between detected change and re-execution, in
+                                      milliseconds [default: 100]
+    -f, --from <APP_MANIFEST_FILE>    The application to watch. This may be a manifest (spin.toml)
+                                      file, or a directory containing a spin.toml file. If omitted,
+                                      it defaults to "spin.toml" [default: spin.toml]
+    -h, --help                        Print help information
+        --skip-build                  Only run the Spin application, restarting it when build
+                                      artifacts change
 ```
 
-The table below outlines exactly which files `spin watch` will monitor for changes depending on how you run the command. `spin watch` uses the configuration found on every component in your application.
+{{ blockEnd }}
 
-| Files                   | `spin watch` monitors for changes              | `spin watch --skip-build` monitors for changes |
-| ----------------------- | ---------------------------------------------- | ---------------------------------------------- |
-| Application manifest    | Yes                                            | Yes                                            |
-| `component.build.watch` | Yes                                            | No                                             |
-| `component.files`       | Yes                                            | Yes                                            |
-| `component.source`      | No (Yes if the component has no build command) | Yes                                            |
+{{ blockEnd }}
```

Comment:
~~I might be misreading this but it looks like you got rid of the "which files `spin watch` watches" section.  Was that intentional?~~ You moved it to the discursive section which is probably a better home - thanks!